src/incubator-chat.html

```diff
@@ -64,6 +65,7 @@
       on-scroll="_triggerLazyLoad">
 
       <slot name="loading-indicator">
+          <vaadin-progress-bar indeterminate style="width: 98%;"></vaadin-progress-bar>
```

Comment:
Please declare a css variable e.g. `--incubator-chat-loading-indicator-width: 98%` and use it to style the slot.
```css
  :host ::slotted(vaadin-progress-bar) {
     width: var(--incubator-chat-loading-indicator-width);
```